js/ui/file_manager/ui.file_manager.files_tree_view.js

```diff
@@ -58,26 +49,40 @@ class FileManagerFilesTreeView extends Widget {
 
         this._filesTreeView = this._createComponent($treeView, TreeViewSearch, treeViewOptions);
 
-        eventsEngine.on($treeView, "click", this._raiseClick.bind(this));
+        eventsEngine.on($treeView, "click", treeViewOptions.onItemClick.bind(this));
```

Comment:
Is it correct? `treeViewOptions.onItemClick` refers to the "onDirectoryClick" action, and I'm not sure we can change the context for this call.